js/samples/quickstart-java/src/main/webapp/index.jsp

```diff
@@ -1,95 +1,106 @@
 <html>
 <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title> - QuickstartSampleWebApp</title>
-        <link rel="stylesheet" href="/resources/bootstrap.css" />
-        <link rel="stylesheet" href="/resources/site.css" />
-        <script type='text/javascript' src='https://contentstorage.onenote.office.net/onenoteltir/immersivereadersdk/immersive-reader-sdk.0.0.3.js'></script>
+    <meta charset='utf-8'>
+    <title>Immersive Reader C# Quickstart</title>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+
+    <script type='text/javascript' src='https://code.jquery.com/jquery-3.3.1.min.js'></script>
+    <script type='text/javascript'
+            src='https://contentstorage.onenote.office.net/onenoteltir/immersivereadersdk/immersive-reader-sdk.0.0.3.js'></script>
```

Comment:
Can this be on line 8 or is there a max line length set somewhere?

Reply:
The editor probably did this automatically, I'll fix